guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "Alice posts a static address on the website of her non-profit, Chasing Rabbits. Bob uses it to send bitcoin from his wallet. This transaction pays a derived on-chain address that Alice can recognise while scanning the blockchain."
+   width = 1024
+   height = 640
+   layout = "full-width"
+%}
+
+## A new conceptual model
+
+Silent payments introduce a new technical model that separates payment information based on their purpose.
+
+For example: the static address is only shared with senders who use it to make payments. Payment detection is done with a separate scan key and doesn't use the static address at all. Here are the main components of this new model:
+
+| Component       |Used by         | Purpose |
+|:-------------|:------------------|:------|
+| Static address | Senders | Making payments
+| Scan key     | Nodes & [light clients](/guide/how-it-works/nodes/#light-nodes)  | Detecting payments
+| Spend key   | Wallet owner | Spending bitcoin
+
+
+The above model introduces a scan key that is used for payment scanning and a static address which is used for deriving on-chain addresses. This is in contrast with the BIP-32 model where [extended public keys](/guide/glossary/#extended-public-key-xpub-ypub-zpub) are used for both purposes. Below is a summary of the differences between the two models:
+
+| BIP-32       | Silent payments          | Benefit  |
+|:-------------|:------------------|:------|
+| Single-use on-chain address           | Static address | Reusable & untraceable on-chain |
+| Extended private key (xprv) | Spend (private) key   |   |
+| Extended public key (xpub)           | Scan (public) key      | Nodes & [light clients](/guide/how-it-works/nodes/#light-nodes) can scan for payments but cannot derive addresses    |
+| *Address-based model*           | *Contact-based model* | *More intuitive and familiar*  |
+
+
+## Labels & contacts
+
+### Labels
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "Adding labels to a static address before posting or sharing with others."
+   width = 250
+   height = 542
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Labels or tags (such as `business` or `no-kyc`) are brief identifiers often associated with information such as payments, transactions and addresses. They're used to organise, categorise or quickly spot items from a large set.
```

Comment:
Is it labels or tags? Are those two different features? If tag is just another word people use for labels sometimes, then I'd make that clear.